tests/basic.js

```diff
@@ -18,6 +18,23 @@ function load() {
     return require(process.cwd());
 }
 
+function testDetail(t, d, uniqueId) {
```

Comment:
so as you said on slack, this should be testMovieDetail, and we should have a testShowDetail